cont.c

```diff
@@ -2080,9 +2080,10 @@ storage_access_must_be_from_same_fiber(VALUE self)
 }
 
 /**
- *  call-seq: Fiber.current.storage -> hash (dup)
+ *  call-seq: fiber.storage -> hash (dup)
```

Comment:
I'm against this change.  Having Fiber.current in call-seq makes it more clear this method can only be called on the current fiber.  Same for `storage=`

Reply:
@jeremyevans I should've explained in GH comments.
The change was introduced because it [renders](https://docs.ruby-lang.org/en/master/Fiber.html#method-i-storage) really weird:
![image](https://user-images.githubusercontent.com/129656/209224529-cf5d76e8-a817-456c-b1c0-04e47e21b167.png)

It is an **instance** method, whose signature is documented as `current.storage`, like, implying that one should do `any_fiber.current.storage`.

Mine renders like this (like any other instance method's signature):
![image](https://user-images.githubusercontent.com/129656/209224855-12484b76-8c0e-4e1c-9e46-3c4aab514d3b.png)
With "it is instance method all right, but can be called only on `Fiber.current` moved to textual part".
I believe it is clearer.
But if you are sure it should be changed back, I'll do.

Reply:
I see.  This looks like an RDoc bug.  It's removing the `Fiber.` like it would for a class method, even though the documentation is for an instance method and such removal makes no sense.  However, since it's probably too late to fix RDoc, I agree with you that it's best to make the changes in the PR.

Reply:
I think it just mechanically removes anything before the first dot (allowing to write `foobar.method` for documenter's pleasure?.. Though, too late to investigate indeed.)